jdl/exporters/jhipster-entity-exporter.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
  * @param passedConfiguration the object having the keys:
  *        - entities: the entities to exporters,
  *        - forceNoFiltering: whether to filter out unchanged entities,
- *        - skipEntityFilesGeneration: whether to skip file write to disk,
```

Comment:
The function is deprecated, I'll be removing them in a PR soon. No need to update it.